src/main/java/flaxbeard/steamcraft/SteamcraftItems.java

```diff
@@ -441,21 +438,29 @@ private static void registerMetals() {
     }
 
     private static void registerFood() {
-
-        steamedFish = new ItemSteamedFood((ItemFood) Items.cooked_fished).setUnlocalizedName("steamcraft:steamedFish").setCreativeTab(Steamcraft.tab);
+        steamedFish = new ItemSteamedFood((ItemFood) Items.cooked_fished)
+          .setUnlocalizedName("steamcraft:steamedFish").setCreativeTab(Steamcraft.tab);
         GameRegistry.registerItem(steamedFish, "steamedFish");
-        ItemFishFood.FishType[] afishtype = ItemFishFood.FishType.values();
         SteamcraftRegistry.addSteamFood(Items.cooked_fished, steamedFish);
 
-        steamedChicken = new ItemSteamedFood((ItemFood) Items.cooked_chicken).setUnlocalizedName("steamcraft:steamedChicken").setCreativeTab(Steamcraft.tab);
+        ItemStack salmonStack = new ItemStack(Items.cooked_fished, 1, 1);
+        steamedSalmon = new ItemSteamedFood((ItemFood) salmonStack.getItem(), salmonStack)
+          .setUnlocalizedName("steamcraft:steamedSalmon").setCreativeTab(Steamcraft.tab);
+        GameRegistry.registerItem(steamedSalmon, "steamedSalmon");
+        SteamcraftRegistry.addSteamFood(Items.cooked_fished, 1, steamedSalmon, -1);
+
+        steamedChicken = new ItemSteamedFood((ItemFood) Items.cooked_chicken)
+          .setUnlocalizedName("steamcraft:steamedChicken").setCreativeTab(Steamcraft.tab);
         GameRegistry.registerItem(steamedChicken, "steamedChicken");
         SteamcraftRegistry.addSteamFood(Items.cooked_chicken, steamedChicken);
 
-        steamedBeef = new ItemSteamedFood((ItemFood) Items.cooked_beef).setUnlocalizedName("steamcraft:steamedBeef").setCreativeTab(Steamcraft.tab);
+        steamedBeef = new ItemSteamedFood((ItemFood) Items.cooked_beef)
+          .setUnlocalizedName("steamcraft:steamedBeef").setCreativeTab(Steamcraft.tab);
         GameRegistry.registerItem(steamedBeef, "steamedBeef");
         SteamcraftRegistry.addSteamFood(Items.cooked_beef, steamedBeef);
 
-        steamedPorkchop = new ItemSteamedFood((ItemFood) Items.cooked_porkchop).setUnlocalizedName("steamcraft:steamedPorkchop").setCreativeTab(Steamcraft.tab);
+        steamedPorkchop = new ItemSteamedFood((ItemFood) Items.cooked_porkchop)
+          .setUnlocalizedName("steamcraft:steamedPorkchop").setCreativeTab(Steamcraft.tab);
         GameRegistry.registerItem(steamedPorkchop, "steamedPorkchop");
         SteamcraftRegistry.addSteamFood(Items.cooked_porkchop, steamedPorkchop);
     }
```

Comment:
This method could use generification by splitting it into a helper method and calling it with that, the basic format of all the additions to this are fairly simple. Otherwise the PR looks good to me